crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -995,12 +992,12 @@ async fn c8y_mapper_syncs_pending_child_alarms_on_startup() {
 
     sm_mapper.abort();
 }
+*/
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+#[tokio::test]
 #[serial]
```

Comment:
Also, why do we need to run `serial` now? We use no longer mock MQTT broker.

Reply:
Yeah, you're right. It doesn't need to be serial anymore. I'll make that change.